DetectorDescription/Core/src/DDCompactView.cc

```diff
@@ -35,19 +35,14 @@ class DDDivision;
 */    
 // 
 DDCompactView::DDCompactView( const DDLogicalPart & rootnodedata )
-  : rep_( std::make_unique<DDCompactViewImpl>( rootnodedata )),
-    worldpos_( std::make_unique<DDPosData>( DDTranslation(), DDRotation(), 0 ))
-{}
-
-DDCompactView::DDCompactView( const DDName& name )
+  : rep_( new DDCompactViewImpl( rootnodedata )),
+    worldpos_( new DDPosData( DDTranslation(), DDRotation(), 0 ))
 {
-  DDMaterial::StoreT::instance().setReadOnly( false );
-  DDSolid::StoreT::instance().setReadOnly( false );
-  DDLogicalPart::StoreT::instance().setReadOnly( false );
-  DDSpecifics::StoreT::instance().setReadOnly( false );
-  DDRotation::StoreT::instance().setReadOnly( false );
-  rep_ = std::make_unique<DDCompactViewImpl>( DDLogicalPart( name ));
-  worldpos_ = std::make_unique<DDPosData>( DDTranslation(), DDRotation(), 0 );
+  DDMaterial::StoreT::instance().setReadOnly(false);
```

Comment:
@fabiocos - this is the only change needed to revert the store lock. I'll submit a new PR for that.

Reply:
@ianna, ok, please do it, so as we may test and integrate it asap